pytests/tests/test_metalink.py

```diff
@@ -38,7 +38,7 @@ def set_metalink(utils, enabled):
         utils.run([ 'sed', '-e', '/metalink/ s/^#*/#/g', '-i', tdnf_repo ])
 
 def set_md5(utils, enabled):
-    photon_metalink = os.path.join(utils.config['repo_path'], 'photon-test/repodata/photon.metalink')
+    photon_metalink = os.path.join(utils.config['repo_path'], 'photon-test/metalink')
```

Comment:
Can we keep 'photon-test/metalink' in a global variable or config json file so that if we change this again changing at one place will suffice?

Reply:
addressed this. Kept global variable in test file.